README.md

```diff
@@ -1 +1,55 @@
-# iabtcf-java
\ No newline at end of file
+
+
+# @iabtcf/core-Java
+
+Ensures consistent of decoding of [IAB's Transparency and Consent Framework (TCF)](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework) [TC Strings](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#about-the-transparency--consent-string-tc-string) and the stateful persistence of the Transparency and Consent information while providing tools for the handling and manipulation of the [TCF](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework) [Global Vendor List](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#the-global-vendor-list) data all free and open sourced ([License](LICENSE)).
+
+  - [Installation](#installation)
+  - [Including in your project](#using)
+  - [TCModel](#tcmodel) - Creates a stateful model to store a Transparency and Consent user interaction with all the fields specified in the [TC string](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#about-the-transparency--consent-string-tc-string) encoding schema.
+     - [Information that is stored in a TC string (specification)](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#what-information-is-stored-in-a-tc-string)
+- [GVL](#gvl) - The [Global Vendor List](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#the-global-vendor-list) contains all of the information about vendors and legal language to display to users in a CMP user interface, this component helps manage it.
+- [TCModelEnum](#tcmodelenum) - The library class for implementing the dynamic getting and setting fields of the [TCModel](#tcmodel) during runtime.
+- [TCString](#tcstring) - Decodes a [TC string](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#about-the-transparency--consent-string-tc-string) into a [`TCModel`](#tcmodel)
+      + [Decode an IAB TC String](#using)
+  
+#### Installation
+- Prerequisites
+    + Java 1.8 
+    + Maven 2.0+
+
+Add the **com.iabtcf** dependency in your project's pom.xml. 
+```xmldo
+<dependency>
+  <groupId>com.iabtcf</groupId>
+  <artifactId>iabtcf-core</artifactId>
+  <version>1.0.4-alpha.0</version>
```

Comment:
This should be 1.0.2-alpha.0

Reply:
Wouldn't it be better to just provide link to maven central? It seems redundant to update the readme every release...

Reply:
it's common practice to include a copy-pasteable link to the latest version of the library in the readme. i havn't read through this in-depth, but seems like updating a README.md with a template that injects the version field can be automated,

https://stackoverflow.com/questions/34629326/update-versions-contained-in-readme-on-maven-release